parsers/lib/validation.py

```diff
@@ -5,13 +5,34 @@
 from logging import getLogger
 
 
+def check_key_is_present(datapoint, key, logger):
```

Comment:
It's more than the presence of a key: it's actually also that the value is not None.
I'd rename to `check_value_is_float`?